From wfp5p@viridian.itc.virginia.edu  Thu Aug 19 10:31:34 2010
From: Bill Pemberton <wfp5p@virginia.edu>
To: linux-usb@vger.kernel.org
Cc: greg@kroah.com
Subject: USB: ssu100: add disconnect function for ssu100
Date: Thu,  5 Aug 2010 17:01:11 -0400
Message-Id: <1281042071-11829-8-git-send-email-wfp5p@virginia.edu>

Add a disconnect function to the functions of this device.  The
disconnect is a call to usb_serial_generic_disconnect() so it requires
that symbol to be exported from generic.c.

Signed-off-by: Bill Pemberton <wfp5p@virginia.edu>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 drivers/usb/serial/ssu100.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/drivers/usb/serial/ssu100.c
+++ b/drivers/usb/serial/ssu100.c
@@ -687,7 +687,6 @@ static void ssu100_process_read_urb(stru
 	tty_kref_put(tty);
 }
 
-
 static struct usb_serial_driver ssu100_device = {
 	.driver = {
 		.owner = THIS_MODULE,
@@ -709,6 +708,7 @@ static struct usb_serial_driver ssu100_d
 	.tiocmset            = ssu100_tiocmset,
 	.ioctl               = ssu100_ioctl,
 	.set_termios         = ssu100_set_termios,
+	.disconnect          = usb_serial_generic_disconnect,
 };
 
 static int __init ssu100_init(void)
